### PR TITLE
Plone Training Guide for Content Editors

### PR DESCRIPTION
**Pull request by tkimnguyen**
This is the training guide we used at the 2024 conference class.

Remaining to do:

- [ ] Rather than refer to screenshots saved on S3, save them to the repo and update the links

Also see issue with other additions to the guide: https://github.com/plone/training/issues/899

----
📚 Documentation preview 📚: https://plone-training--896.org.readthedocs.build/

### CODE REVIEW

**Review by ericof — APPROVED** [review]

File: docs/content-editing/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Plone is a fully featured web content management system for creating public websites and intranets. It lets you create and organize pages, folders, files, news items, events, almost any type of information you can imagine.
+
+Noteworthy feature of Plone include:
+- collaborative editing: support for having multiple editors creating, editing, and managing a website simultaneously
+- workflow: content items, when created, are initially in a private state and are not visible to the public, and can be advanced to other states, such as "review" and "published"
+- content rules: sets of triggers and actions that can be performed automatically on content items
+
+In this user guide, we will show how to use Plone to create, edit, and manage content.
+
+### For Attendees of the Plone Conference Training
```
Comment (stevepiercy) [inline docs/content-editing/index.md:29]: Decrease heading level

```suggestion
## For Attendees of the Plone Conference Training
```

File: docs/content-editing/add-a-file.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+# Add a File
+
+You can add any type of file to your Plone site. Files can be any "binary file", including PDFs, text files, videos, and Word documents. 
+
+You could follow this method to add an image, but rather than add an image as a file, you should add an image as an image! (see Add an Image below).
+
+## Add a File
+
+[//]: # (#### [Made by T. Kim Nguyen with Scribe]&#40;https://scribehow.com/shared/Add_a_File__Yh-Avk1hQwatZwoP5HAGkA&#41;)
```
Comment (stevepiercy) [inline docs/content-editing/add-a-file.md:18]: Use proper comment syntax. `[//]: #` causes the following warning. Globally replace with `%`. This change is mandatory to remove noisy warnings during builds.

```console
docs/content-editing/add-a-link.md:18: WARNING: Duplicate reference definition: // [myst.duplicate_def]
```

```suggestion
% (#### [Made by T. Kim Nguyen with Scribe]&#40;https://scribehow.com/shared/Add_a_File__Yh-Avk1hQwatZwoP5HAGkA&#41;)
```

**Review by spereverde — APPROVED** [review]
Nicely written, thanks for writing all that up!

File: docs/content-editing/add-a-file.md (hunk 2 above)
Comment (tkimnguyen) [reply docs/content-editing/add-a-file.md]: somehow I wasn't able to get that % syntax to work, so I deleted all the commented lines...

File: docs/content-editing/add-a-file.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+
+11\. You can select one or more related items by navigating and by searching the site.
+
+In this example, we select as a related item the page "Content Types" by clicking on its icon.
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2024-11-16/06209fcc-c929-4ca1-8b29-11f4d490b14f/ascreenshot.jpeg?tl_px=800,149&br_px=2520,1110&force_format=jpeg&q=100&width=1120.0&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=612,277)
+
+
+12\. Click the X icon to exit from the related items selection widget.
```
[the remark this answers is not in the document]
Comment (tkimnguyen) [reply docs/content-editing/add-a-file.md]: Thanks Steve. I'm starting to change those remote images to local ones and am adding the enhanced view with the cards.

File: docs/content-editing/add-a-file.md (hunk 2 above)
Comment (stevepiercy) [reply docs/content-editing/add-a-file.md]: What did you try that did not work? The suggestion works for me. See https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#comments

**Pull request merged**